### PR TITLE
fix issue 285

### PR DESCRIPTION
**Pull request by fgrunewald**
@csbrasnett this is a clean PR with all changes we did yesterday. I also fixed the template generation issue. Can you test this branch?

To Do
- [x] add test for volume computation
- [x] make sure IDP parameters are correct
- [x] add citation to IDP paper
- [x] add warning to include new martini itp file??
- [x] write user tutorial

### CODE REVIEW

**Comment by csbrasnett** [comment]
Bonded parameters look good on testing, but the VS-BB exclusions don't seem to be generated at all?

for generating coordinates the molecule checking fails too, error of two disjointed parts

**Comment by fgrunewald** [comment]
@csbrasnett I fixed the exclusions. For the structure coordinate generation works though? Can you make a example available to me.

**Comment by csbrasnett** [comment]
Looks to work now

**Comment by fgrunewald** [comment]
@csbrasnett the peprint is already out. Are these parameters up to date? Can we merge and release it?

File: polyply/src/minimizer.py [hunk 1]
```diff
@@ -183,7 +183,8 @@ def target_function(positions):
                 energy += new
         return energy
 
-    opt_results = scipy.optimize.minimize(target_function, positions, method='L-BFGS-B',
+    opt_results = scipy.optimize.minimize(target_function, np.ravel(positions),
+                                          method='L-BFGS-B',
```
Comment (fgrunewald) [inline polyply/src/minimizer.py:187]: @csbrasnett can you remove the np.ravel from this line? Instead put the following in line 169:
```
positions = np.ravel(np.array(list(coords.values())))
```

**Review by fgrunewald — COMMENTED** [review]
One small nitpick about the minimizer, the rest looks fine. 

However, it seems @pckroon needs to approve this PR.

**Review by pckroon — APPROVED** [review]

**Pull request merged**